### ISSUE TITLE
Would you please explain all the experiment names in your code?

### ISSUE THREAD

**Issue by beyondguo** [XiangLi1999/Diffusion-LM#28]
Hi, thanks for your interesting work!

There are a lot of arguments for `model_args.experiment`:
- 'e2e-tgt-tree', 'e2e-tgt-gen-tree', 'e2e-tgt-gen-spans'
- e2e-tgt-gen-length
- 'e2e-tgt-pos', 'e2e-tgt-gen-pos'
- 'synth_emb', 'pos_emb', 'roc_emb', 'simple-wiki_emb', 'e2e-tgt_emb'
- ....

These choices make it hard to follow your work and reproduce some experiments. I guess lots of choices are for baseline implementation. Which experiments should I choose if I want to run the controlling tasks by Diffusion-LM in your paper? Would you please explain them in your README page? 

Thanks a lot!

**Comment by lcy5058** [comment]
+1

**Comment by radi-cho** [comment]
+1

**Comment by gcdYIKASFVGUSI** [comment]
> 

+1
